### DISCUSSION TITLE
Product Enquiry

### DISCUSSION THREAD

**Discussion by skizo101** [community/community#64607] [New to GitHub]
### Select Topic Area

Question

### Body

Hi. I am relatively new to Github. My start-up wants to organise a hackathon for 50-100 users and I'm completely lost on what package to sign up for. I have little tech experience and would like someone to help me please.

**Comment by LinukaAr** [comment]
Organizing a hackathon on GitHub can be a great way to engage developers and foster innovation. The type of GitHub plan you'll need depends on the specific requirements of your hackathon and how you plan to use GitHub during the event. Here are a few options to consider:

1. **Free Plan (GitHub Free):** If your hackathon is relatively small and you don't anticipate a need for private repositories, you can start with GitHub's free plan. This plan offers public repositories, basic issue tracking, and collaboration tools. Participants can create their own GitHub accounts and collaborate on open-source projects.

2. **Team Plan (GitHub Team):** If you need some level of access control and security, GitHub Team might be suitable. This plan allows you to create private repositories, manage access permissions, and collaborate more effectively. You can invite team members and participants, and you'll have a better way to organize your projects.

3. **Enterprise Cloud Plan (GitHub Enterprise Cloud):** For larger hackathons or if you need more advanced features like SAML-based single sign-on (SSO), advanced auditing, and security features, GitHub Enterprise Cloud could be a consideration. This plan provides an integrated environment for collaboration and coding in a secure and customizable way.

4. **Enterprise Server Plan (GitHub Enterprise Server):** If you require complete control over your hackathon environment and want to host GitHub on your own servers, GitHub Enterprise Server might be the choice. This option is suitable for larger organizations that need high-level customization and have specific security requirements.

5. **GitHub Education:** If your hackathon is focused on educational institutions or students, GitHub Education offers special benefits for educational purposes. You can apply for the GitHub Education program, which provides free access to various GitHub tools and features.

Here are some steps to help you decide:

1. **Define Hackathon Goals:** Consider the goals and scale of your hackathon. Do you need private repositories? Will you require access control? Are you looking for a more integrated and secure environment?

2. **Estimate Participants:** Determine the number of participants you're expecting. Smaller hackathons might be fine with free or lower-tier plans, while larger ones might need more advanced features.

3. **Privacy and Security:** Consider the privacy and security requirements of your participants. If you're dealing with sensitive data, you might need a plan with advanced security features.

4. **Ease of Use:** As someone new to GitHub, consider the ease of use of the plan you choose. Make sure it aligns with your comfort level and technical skills.

5. **Budget:** Check the pricing details for each plan to ensure it fits within your budget. GitHub offers various pricing tiers, so find the one that offers the best value for your hackathon's needs.

Before you make a decision, I recommend reaching out to GitHub's sales or support teams. They can provide more tailored guidance based on your specific requirements and answer any questions you might have. Additionally, exploring the GitHub documentation and resources for event organizers can provide you with valuable insights into setting up and managing your hackathon on GitHub.
